README.md

```diff
@@ -114,9 +118,37 @@ BENCHMARK(BM_StringCopy);
 BENCHMARK_MAIN();
 ```
 
-Don't forget to inform your linker to add benchmark library e.g. through `-lbenchmark` compilation flag.
+Don't forget to inform your linker to link to the benchmark library e.g. through `-lbenchmark` compilation flag, and also the pthread library ('-lpthread'). If running google benchmarks on Windows, the shlwapi library must additionally be linked to (`-lshlwapi`) and the library location specified eg. ../../libraries/google_benchmark/lib. If running Solaris, kstat (`-lkstat`) must be linked to.
```

Comment:
These a problems, not intended behaviour.
I think this should stay as is in the section at the end.

Reply:
They may be problems, but the user is not going to see the bottom of the document when they are trying to compile the first time, following the guidelines. When corrected in Gbenchmark, they can be corrected here also.

Reply:
BTW, I spent the better part of an hour googling to find the answer above the first time I tried to compile google benchmark code on windows. So I'm speaking from experience, not theory. You can't expect the user to read the whole document in one go, or assume that parts which relate to linking are somewhere else for some previously unknown reason.